packages/cli/index.ts

```diff
@@ -115,14 +115,14 @@ const main = defineCommand({
 
           for (const p of paths) {
             const pJsonPath = path.resolve(p, "package.json");
-            const { name } = await importPackageJson(pJsonPath);
+            const { name } = await readPackageJSON(pJsonPath);
 
-            if (compact) {
+            if (compact && name) {
```

Comment:
let's check the name in another condition

Reply:
Done 🙌